Source/CarthageKit/Swift3Shims.swift

```diff
@@ -134,6 +134,27 @@
 		}
 	}
 
+	public typealias Scanner = NSScanner
```

Comment:
This must be `public` because of this: https://github.com/Carthage/Carthage/pull/1595/files#diff-9a71f3c0c991f37b53338e11f4d93de2R18